examples/compiled/interactive_concat_layer.vg.json

```diff
@@ -255,7 +255,7 @@
           "grid": false,
           "title": "Rotten_Tomatoes_Rating (binned)",
           "labelOverlap": true,
-          "tickCount": {"signal": "ceil(concat_0_height/20)"},
+          "tickCount": {"signal": "ceil(concat_0_height/10)"},
```

Comment:
I don't think this is better

<img width="918" alt="screen shot 2018-12-28 at 22 09 36" src="https://user-images.githubusercontent.com/589034/50534136-4d2f8e80-0aed-11e9-9c64-c0d47db2e9f6.png">